### PR TITLE
Fix package name

### PR DESCRIPTION
**Pull request by xylar**
It should be "scrapbook", not "scrapbook-build"

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
This was not correctly reverted in #8

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
```
=================================== FAILURES ===================================
_______________ test_unsupported_arrow_conversions[test_input0] ________________

test_input = Scrap(name='foo', data=          foo
0  {bar, foo}, encoder='pandas', display=None)

    @pytest.mark.parametrize(
        ("test_input",),
        [
            # Sets can't convert
            (
                Scrap(
                    name="foo",
                    data=pd.DataFrame(data={"foo": pd.Series([{"foo", "bar"}], dtype='object')}),
                    encoder="pandas",
                ),
            ),
        ],
    )
    def test_unsupported_arrow_conversions(test_input):
>       with pytest.raises(pyarrow.lib.ArrowInvalid):
E       Failed: DID NOT RAISE <class 'pyarrow.lib.ArrowInvalid'>

../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/lib/python3.10/site-packages/scrapbook/tests/test_encoders.py:250: Failed
```

I'm afraid this is getting too far into the weeds for me. I thought this would be a simple name change but it looks like testing has atrophied in the last year.

**Comment by xylar** [comment]
Feel free to push a fix to this branch if you have one.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,6 +29,8 @@ test:
   requires:
     - coverage
     - ipython
+    # seemingly needed for ipython_genutils
+    - jupyter
```
Comment (bollwyvl) [inline recipe/meta.yaml:33]: no, you don't want to depend on `jupyter`: it hauls in `qt` and a lot of other stuff...

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% set version = "0.5.0" %}
 
 package:
-  name: scrapbook-build
+  name: scrapbook
```
Comment (bollwyvl) [inline recipe/meta.yaml:4]: good catch.

File: recipe/meta.yaml (hunk 1 above)
Comment (xylar) [reply recipe/meta.yaml:33]: This is just in testing.  If you have another fix, please do it instead.

**Comment by bollwyvl** [comment]
Thanks, fixed this up over on #10

**Pull request closed without merging**